src/SourceBuild/content/repo-projects/source-build-reference-packages.proj

```diff
@@ -19,13 +19,19 @@
     <BuildArgs>$(BuildArgs) /p:LocalNuGetPackageCacheDirectory=$(LocalNuGetPackageCacheDirectory)</BuildArgs>
   </PropertyGroup>
 
+  <ItemGroup>
+    <BuiltSdkPackageOverride Include="@(NoTargetsSdkOverride)" />
+    <BuiltSdkPackageOverride Include="@(TraversalSdkOverride)" />
```

Comment:
Where in SBRP is the Microsoft.Build.Traversal msbuild sdk used?

Reply:
It isn't used. `BuiltSdkPackageOverride` items are used for extracting toolsets (SDKs) after repo's build: https://github.com/dotnet/installer/blob/de345f1baaa42816bf4f9c40eab2e57cc1f1fb58/src/SourceBuild/content/repo-projects/Directory.Build.targets#L568